pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -292,6 +310,14 @@ func (state PipelineRunState) GetFinalTasks(d *dag.Graph, dfinally *dag.Graph) [
 	return tasks
 }
 
+// Check if a PipelineTask belongs to the specified Graph
+func isTaskInGraph(n string, d *dag.Graph) bool {
```

Comment:
"n" is not a very informative name, since the type it's a string, perhaps we could use something like "pipelineTaskName"?

Reply:
Done